xpano/gui/panels/about.h

```diff
@@ -1,21 +1,24 @@
 #pragma once
 
+#include <future>
 #include <vector>
 
 #include "utils/text.h"
+using Texts = std::vector<xpano::utils::Text>;
```

Comment:
It is not a good idea to put such declarations in the global namespace, this can lead to name clashes and weird bugs later.

Please put the using declaration inside the namespace in the text.h file, right next to the Text struct and then refer to it as utils::Texts

Reply:
Some details: https://google.github.io/styleguide/cppguide.html#Namespaces